### PR TITLE
feat: Add hyvor component to context

### LINKED ISSUE #21: Add comment section to context (hyvor?)

### PR DESCRIPTION
**Pull request by katjam**
Fixes #21

## Description

- Add hyvor script and component to context section
- Need to test with a whitelisted url

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <div id="app">
       <div></div>
     </div>
+    <script async src="https://talk.hyvor.com/embed/embed.js" type="module"></script>
```
Comment (katjam) [inline index.html:14]: never sure if I should put scripts in body or head...

**Comment by katjam** [comment]
Merging to test (assuming @stuartleech has whitelisted https://bi4c8d.netlify.app)

**Comment by katjam** [comment]
... have not yet whitelisted:
image

**Comment by stuartleech** [comment]
Screenshot 2024-08-10 at 16 20 54

**Pull request merged**